### ISSUE TITLE
Android - unload/reloading textures result in black rectangles instead of text

### ISSUE THREAD

**Issue by tallavi** [openframeworks/openFrameworks#3555]
The discussion started here: http://forum.openframeworks.cc/t/android-unload-reloading-textures-result-in-black-rectangles-instead-of-text/18038 

Text on native UI elements turns to black rectangles after minimizing and restoring. See attached screenshots. It's directly related to loading/unloading textures in of, but I'm not sure how.
I've managed of creating a super simple reproduction based on "android empty example" that shows the problem!

The issue is reproduced on ALL Android phones I tried, with different OS versions.

The forum post has an attachment of an extremely simple reproduction project. Even though it's only a few lines of code, I couldn't figure out what (if at all) I'm doing wrong, and it's hard for me to understand what's going on inside (since I cannot debug it). 

So I'm suspecting a bug related to how of unload/reloads the textures.

Thanks.

![9badaa551bbb1786](https://cloud.githubusercontent.com/assets/3534478/5721179/f4e1f1ca-9b36-11e4-979b-da5d53dae732.png)
![277d63c378bf14e4](https://cloud.githubusercontent.com/assets/3534478/5721180/f4e54a32-9b36-11e4-8743-18f36bd350d9.png)

**Comment by arturoc** [comment]
yeah that's really crazy, i can't understand how anything in the GL context can change the gui elements at all. i'll try to check your example when i have a moment

**Comment by tallavi** [comment]
Thanks allot! I'm going crazy over here!

**Comment by danthemellowman** [comment]
Can you share the version of your NDK & Android SDK Platform & Build Tools
you are building with?

I'm going to try your sample out on my phones.  What devices/versions did
you try this on?

**Comment by tallavi** [comment]
Hey @danthemellowman,

I have it reproducing on HTC M8 with Stock 4.4.3 and Samsung s3 (i9300) with cyanogenMod 4.3.1.

I have a few other devices not currently on me, some cheap samsung and a Xiomi, all reproducing.

I'm pretty confident that you can just build with whatever setup and device you have, but if you want my specific configuration, I'm on latest OSX yosemite, building with Android SDK 4.4.2, and using NDK r9c.     

Eclipse IDE for Android developers and ADT plugin, version 23.0.2.1259578.

Any other information you need?

**Comment by danthemellowman** [comment]
Thanks for the info!

**Comment by tallavi** [comment]
My pleasure! Have you gotten this reproduced?
